docker/Dockerfile

```diff
@@ -45,12 +47,14 @@ RUN ln -s /opt/tegola_src /opt/tegola_src/src/github.com/go-spatial/tegola
 # Build binary
 WORKDIR /opt/tegola_src/cmd/tegola
 ENV GOPATH=/opt/tegola_src
+RUN go get -u github.com/theckman/goconstraint/go1.8/gte
```

Comment:
Ideally we should vendor this rather than add it here.

Reply:
I looked into it and this package is [vendored](https://github.com/go-spatial/tegola/blob/master/vendor/vendor.json#L407). Is the Docker build failing without this line?

Reply:
Yes it states:
```
Step 12/19 : RUN go build -v -gcflags "-N -l" -o /opt/tegola
 ---> Running in 662cda042194
main.go:7:2: cannot find package "github.com/theckman/goconstraint/go1.8/gte" in any of:
        /usr/local/go/src/github.com/theckman/goconstraint/go1.8/gte (from $GOROOT)
        /opt/tegola_src/src/github.com/theckman/goconstraint/go1.8/gte (from $GOPATH)
```

Reply:
Strange. Can you check your local copy. I see this in the vendor directory [here](https://github.com/go-spatial/tegola/tree/v0.8.x/vendor/github.com/theckman/goconstraint/go1.8/gte) and I would prefer to keep this go get request out of the Docker image.